### ISSUE TITLE
Discuss Best Practices for Accessibility Handling 

### ISSUE THREAD

**Issue by bischoffa** [department-of-veterans-affairs/va-mobile-app#7183]
**Overview:** 
As Ad Hoc Mobile, we want to set  internal guidelines and boundaries on accessibility/usability in order to know when there is a test finding if its a bug (within our guidelines) or and edge case (outside of our guideline). 

**Info:** 
- Derived off a conversation on 11/1 discussing a bug ticket [5106](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/5106)
- Intent is to help identify what are truly bugs vs non bugs that are accepted for the time being given its outside of our guidelines

**Acceptance Criteria:** 

- [x] Brea to schedule a meeting with FE, Accessibility, QA 
- [x] FE, Accessibility, QA group to review [Accessibility Test Plan](https://docs.google.com/document/d/1fFh4I7CTNW8YZRyHBW5vPX0EQhB3S2ZmyamipXPwGfY/edit#heading=h.jqzcoj4fqy86)
- [x]  Group discusses and identifies internal guidelines Mobile wants to set
- [x]  PM to review group's proposal and develops documentation/process to track and capture

**Comment by Sparowhawk** [comment]
@brea11y looking at the document you linked the only thing that I can think of off the top of my head that we cannot support completely is display zoom due to the limitations I discussed. I do not know at what point display zoom breaks. Maybe @TKDickson can help with that one

**Comment by bischoffa** [comment]
@brea11y can you add a point estimate to this ?
